portal/src/main/scala/cromwell/pipeline/service/UserManagementService.scala

```diff
@@ -0,0 +1,19 @@
+package cromwell.pipeline.service
+
+import cromwell.pipeline.datastorage.dao.repository.UserRepository
+import cromwell.pipeline.datastorage.dto.UserId
+import cromwell.pipeline.datastorage.dto.management.DeactivateUserRequestByEmail
+import cromwell.pipeline.utils.auth.AuthUtils
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class UserManagementService (userRepository: UserRepository, authUtils: AuthUtils)(implicit executionContext: ExecutionContext){
+  def deactivateByEmail(request: DeactivateUserRequestByEmail): Future[Int] = {
+    userRepository.deactivateUserByEmail(request.email)
+  }
```

Comment:
Why related methods in Service and Repository have different names (deactivateByEmail and deactivateUserByEmail)?